WordPress/src/main/java/org/wordpress/android/ui/prefs/WPIconWidgetPreference.java

```diff
@@ -55,9 +58,15 @@ protected void onBindView(@NonNull View view) {
             button.setOnClickListener(new View.OnClickListener() {
                 @Override
                 public void onClick(View v) {
-                    getOnPreferenceClickListener().onPreferenceClick(wpButtonPreference);
+                    getOnPreferenceClickListener().onPreferenceClick(iconWidgetPreference);
                 }
             });
         }
+
+        if (view.findViewById(R.id.domain) != null) {
```

Comment:
`WPIconWidgetPreference` feels like a general widget to me, whereas we are using `R.id.domain` and setting the text to the url. I am wondering if it would be better to either keep this widget general or rename it to something more specific.